CHANGELOG.md

```diff
@@ -2,10 +2,18 @@
 
 ## [Unreleased]
 
+## [2.0.3] - 2018-06-28
+
+### Changed
+* Reverted proper destroy (#34) (@ralphtheninja)
+
+**Historical Note** @ralphtheninja was trigger happy and released `v2.0.2` incorrectly, which should only have included a fix to node 4.
```

Comment:
No need to take the blame (we all love you to bits), IMO just say "**Historical Note** The previous release was meant to restore node 4 and included an additional change by mistake."